roles/kafka_connect_replicator/tasks/rbac_replicator.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Get Authorization Token for Connect Replicator ERP User
+  import_role:
+    name: common
+    tasks_from: get_authorization_token.yml
+  vars:
+    oauth: "{{ oauth_enabled }}"
```

Comment:
shouldn't this be kafka_connect_replicator_erp_oauth_enabled?

Reply:
yes agreed